src/Command/BaseCommand.php

```diff
@@ -29,7 +30,7 @@
 abstract class BaseCommand extends Command
 {
     /**
-     * @var ManagerInterface
+     * @var ManagerInterface&ClearableManagerInterface
```

Comment:
I'm not sure the `ClearableManagerInterface` is required.

When using `clear, we should just make an `instanceof` check before calling clear.
This is what we did on master.